### DISCUSSION TITLE
Can't sign up for the student pack

### DISCUSSION THREAD

**Discussion by Lbpereira** [community/community#137425] [GitHub Education]
When I try to sign up for the pack, my school name is aked and when I write it down it says I need to add and berify my student email adress related to my school and then refresh the page. 
I did it 5 times and it still doesn't allow me to move on to the next step....

**Comment by alx-mp4** [comment]
Hi @Lbpereira, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
